website/content/en/docs/tasks/operator.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Deployment Operator
```

Comment:
by default, the URL in the browser would be tasks/operator.
If you want to control it (e.g., tasks/operator-install), I think you can using some frontmatter fields.
For example, [slug](https://gohugo.io/content-management/urls/)